### DISCUSSION TITLE
and just how to update

### DISCUSSION THREAD

**Discussion by syswombat** [lvjr/tabularray#267] [Q&A]
hello
i have just a small and maybe stupid question.
But how do i update only the tabularray packet to the newest ?
starting the minimal installation i got a 

! LaTeX Error: Environment tblr undefined.

have a nice day
vinc

TeXstudio 3.0.4 (Build: 3.0.4+ds-1)
Nutze Qt-Version 5.15.2, kompiliert mit Qt 5.15.2 R
Compiler ist LuaLaTex

Distro: Debian GNU/Linux 11 (bullseye)
Kernel: 5.18.x  amd64 x86_64 oder neuer

**Comment by lvjr** [comment]
Download `tabularray.sty` from https://ctan.org/pkg/tabularray and put it in the folder of your tex document.

**Reply by syswombat** [reply]
@lvjr thanks for your feedback
downloaded and renamed it - but have this file really to be put where the tex document is?
because now i got other error about a File `ninecolors.sty' not found. \begin{document}
normaly i use \usepackage[x11names]{xcolor}

**Reply by lvjr** [reply]
Download `ninecolors` package here: https://www.ctan.org/pkg/ninecolors
